src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  isActive?: boolean,
+  selectedTodosId?: number[],
+  id: number,
+};
+
+export const Loader: React.FC<Props> = ({
```

Comment:
loader component shouldn’t know about selectedTodosId and id. I believe that isActive enough 
```suggestion
type Props = {
  isActive?: boolean,
};

export const Loader: React.FC<Props> = ({
```

Reply:
Ok 🤔